clusterloader2/pkg/measurement/common/slos/windows_node_resource_usage.go

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
You need to rebase - otherwise it's much harder to review since it's unclear what is new in this PR.

Reply:
sorry i had rebased but probably against the wrong branch. this has been removed.